release.config.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+    branches: ['master'],
+    tagFormat: 'v${version}',
+    repositoryUrl:
+        'https://github.com/mparticle-integrations/mparticle-javascript-integration-criteo',
+    plugins: [
+        [
+            '@semantic-release/commit-analyzer',
+            {
+                preset: 'angular',
+                releaseRules: [
+                    { type: 'feat', release: 'minor' },
+                    { type: 'ci', release: 'patch' },
+                    { type: 'fix', release: 'patch' },
+                    { type: 'docs', release: 'patch' },
+                    { type: 'test', release: 'patch' },
+                    { type: 'refactor', release: 'patch' },
+                    { type: 'style', release: 'patch' },
+                    { type: 'build', release: 'patch' },
+                    { type: 'chore', release: 'patch' },
+                    { type: 'revert', release: 'patch' },
+                ],
+            },
+        ],
+        [
+            '@semantic-release/release-notes-generator',
+            {
+                preset: 'angular',
+            },
+        ],
+        [
+            '@semantic-release/changelog',
+            {
+                changelogFile: 'CHANGELOG.md',
```

Comment:
I may be wrong here, but do you need to include `CHANGELOG.md` manually, or will this be generated for you the first time?

Reply:
Also, can you please include `CONTRIBUTING.md`?